htmd/parameterization/cli.py

```diff
@@ -606,20 +656,21 @@ def main_parameterize(arguments=None):
 
         # Set parameters for the fake QM
         if args.fake_qm:
-            qm._parameters = parameters
+            assert not args.nnp
```

Comment:
Remove the assertion. You already have one above so it will never trigger

Reply:
I prefer to keep it, to be safe in future refactoring.